esp32-controller/src/env-example

```diff
@@ -17,5 +17,7 @@
 // -------------------- ENVIRONMENTS
 #define WIFI_SSID ("")
 #define WIFI_PASSWORD ("")
+#define WEBSOCKET_URI ("ws://websocket.org")
```

Comment:
Isn't it better to use an example from localhost?

Reply:
Yes, thanks for noticing